src/reorderable/ReorderableListComponent.coffee

```diff
@@ -23,6 +24,7 @@ class ReorderableListComponent extends React.Component
     @state = {
       initialOrder: order
       order: order
+      listId: if @props.listId then @props.listId else id = uuid.v4()
```

Comment:
Don't forget to handle in componentWillReceiveProps too! listId might be set later. Also, why `id = uuid.v4()` and not just `uuid.v4()`